javaparser-core-testing/src/test/java/com/github/javaparser/utils/CodeGenerationUtilsTest.java

```diff
@@ -54,4 +61,23 @@ void testGetterToPropertyName() {
         assertEquals("Unexpected getterName 'value'", thrown.getMessage());
     }
 
+    @Test
+    void testPackageToAbsolutePath() throws IOException {
+        final Path pathToZip = classLoaderRoot(CodeGenerationUtilsTest.class)
+                .resolve("com/github/javaparser/utils/MyActivity.zip");
```

Comment:
What is this file? If you need a compressed file for your test case there are already resources available for example in the source_zip directory

Reply:
Sure, will replaced